packages/react/src/components/Popover/PopoverContent.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { forwardRef } from 'react';
+
+export type PopoverContentProps = {
+  children: React.ReactNode;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+export type PopoverContentType = React.ForwardRefExoticComponent<
+  PopoverContentProps & React.RefAttributes<HTMLDivElement>
+>;
+
+export const PopoverContent: PopoverContentType = forwardRef(

```

Comment:
Maybe just remove `forwardRef` here as I don't think is needed, and its less refs for react to track.